vocabularies/Analytics.xml

```diff
@@ -167,6 +167,44 @@
         </Property>
       </ComplexType>
 
+      <Term Name="UpdateAction" Type="Analytics.UpdateActionType" Nullable="false">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Common.IsInstanceAnnotation" />
+        <Annotation Term="Core.Description" String="Action for updating an aggregate entity" />
+        <Annotation Term="Core.LongDescription">
+          <String>Transient entities that result from the application of a transformation sequence represent an
+          aggregate of persistent entities (for example, aggregate sales per country and product, as in [OData-Aggr, example 19]).
```

Comment:
Will `[OData-Aggr, example 19]` become a real link once the next spec draft is out?

Reply:
Yes.